.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+# Java Gradle CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-java/ for more details
+#
+version: 2
+jobs:
+  build:
+    branches:
+      only:
+        - master
+        - /feature\/ci.*/
+    docker:
+      - image: circleci/node:10-browsers-legacy
+        environment:
+          MAX_HEAP_SIZE: 2048m
+          HEAP_NEWSIZE: 512m
+
+    working_directory: ~/web-client-sdk-base
+
+    steps:
+      - checkout
+      # Download and cache dependencies.
+      - restore_cache:
+          keys:
+            - dependency-cache-{{ checksum "package-lock.json" }}
+
+      - run:
+          command: npm install
```

Comment:
Does image `circleci/node:10-browsers-legacy` have yarn installed?
If it has, I guess using yarn is better since we use yarn.lock to fix the versions of dependencies.

Reply:
Ah, if we also commit package-lock.json, then it will be fine.